python/cudf/cudf/tests/test_parquet.py

```diff
@@ -667,6 +729,7 @@ def test_parquet_reader_list_large_mixed(tmpdir):
     assert_eq(expect, got, check_dtype=False)
 
 
+"""
```

Comment:
Is this a temporary change that you commented this out to speed up your local development in testing?